package.json

```
@@ -57,5 +64,20 @@
     "@storybook/addons": "^5.3.13",
     "@storybook/preset-create-react-app": "^1.5.2",
     "@storybook/react": "^5.3.13"
+  },
+  "lint-staged": {
```

```suggestion
  "lint-staged": {
    "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
       "prettier --write"
    ]
},
```

can you replace the entire lint-staged config to the suggestion above?